exporter/stackdriver/trace_proto.go

```diff
@@ -163,26 +173,51 @@ func copyAttributes(out **tracepb.Span_Attributes, in map[string]interface{}) {
 	}
 	var dropped int32
 	for key, value := range in {
-		av := tracepb.AttributeValue{}
-		switch value := value.(type) {
-		case bool:
-			av.Value = &tracepb.AttributeValue_BoolValue{BoolValue: value}
-		case int64:
-			av.Value = &tracepb.AttributeValue_IntValue{IntValue: value}
-		case string:
-			av.Value = &tracepb.AttributeValue_StringValue{StringValue: trunc(value, maxAttributeStringValue)}
+		av := attributeValue(value)
+		switch key {
+		case httptrace.URL:
+			(*out).AttributeMap[labelHTTPURL] = &av
```

Comment:
Looks iffy as we are repeating the pointer creation in every case, perhaps we can reverse this and do it once e.g
```go
func attributeValue(v interface{}) *tracepb.AttributeValue
```

and then we don't have to take the address in every spot.